### ISSUE TITLE
Bedrock missing

### ISSUE THREAD

**Issue by EvilOlaf** [WorldCretornica/PlotMe-Core#83]
The bedrock below the plotworld is missing or better said the first layer is dirt. The plotworld has been generated with 0.15.0, therefore I don't know if that has been fixed already.
When I reset or clear a plot the bedrock appears.

Here is the look from below the world: http://www0.xup.to/exec/ximg.php?fid=90071697

```
worlds:
  city:
    PlotFloorBlock: '2'
    XTranslation: 0
    RoadAltBlock: '13'
    RoadMainBlock: '43'
    GroundHeight: 60
    FillBlock: '3'
    ZTranslation: 0
    WallBlock: '44:5'
    PathWidth: 7
    ForSaleWallBlock: '44:1'
    ProtectedWallBlock: '44:4'
    AuctionWallBlock: '44:1'
    PlotSize: 51
```

**Comment by MattBDev** [comment]
I believe I fixed this in 0.15.2 generation. If not, let me know.

**Comment by EvilOlaf** [comment]
Yep, is fixed.
